src/ConversationData.res

```diff
@@ -5,9 +5,9 @@
 let apiBaseUrl = vite_api_url
 
 type rating =
-  | Green
-  | Yellow
-  | Red
+  | @as("green") Green
```

Comment:
This is a fun new feature in ReScript 11. This will map the runtime representation of a variant to a string literal. So `Green` will be represented as `green` in JS.

This is huge, since we now can use nominally typed variants without the need of any encoding / decoding when interacting with the outside world (data fetching, JS libraries, etc).

Until the docs land, here's the full context via our blog post: https://rescript-lang.org/blog/improving-interop